recipes/aws-sdk-cpp/all/test_package/conanfile.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class AwsSdkCppTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            if self.settings.build_type == "Release":
+                self.settings.compiler.runtime = "MT"
+            else:
+                self.settings.compiler.runtime = "MTd"
```

Comment:
Why are you forcing the runtime to be static?

Reply:
When I remove this, I have a link error when creating the package on Windows:

```
-- Build files have been written to: C:/Users/marti/dev/clone/conan-center-index/recipes/aws-sdk-cpp/all/test_package/build/ecbfb49a4e2ea8618319be5f9e02c8e0621391fb

----Running------
> cmake --build C:\Users\marti\dev\clone\conan-center-index\recipes\aws-sdk-cpp\all\test_package\build\ecbfb49a4e2ea8618319be5f9e02c8e0621391fb --config Release -- /m:8 /verbosity:minimal
-----------------
Microsoft (R) Build Engine version 16.7.0+b89cb5fde for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Checking Build System
  Building Custom Rule C:/Users/marti/dev/clone/conan-center-index/recipes/aws-sdk-cpp/all/test_package/CMakeLists.txt
  example.cpp
aws-cpp-sdk-core.lib(ub_core.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MT_StaticRelease' doesn't match value 'MD_DynamicRelease' in example.obj [C:\Users\marti\dev \clone\conan-center-index\recipes\aws-sdk-cpp\all\test_package\build\ecbfb49a4e2ea8618319be5f9e02c8e0621391fb\example.vcxproj]
aws-cpp-sdk-core.lib(HttpTypes.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MT_StaticRelease' doesn't match value 'MD_DynamicRelease' in example.obj [C:\Users\marti\d ev\clone\conan-center-index\recipes\aws-sdk-cpp\all\test_package\build\ecbfb49a4e2ea8618319be5f9e02c8e0621391fb\example.vcxproj]
aws-cpp-sdk-core.lib(Scheme.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MT_StaticRelease' doesn't match value 'MD_DynamicRelease' in example.obj [C:\Users\marti\dev\ clone\conan-center-index\recipes\aws-sdk-cpp\all\test_package\build\ecbfb49a4e2ea8618319be5f9e02c8e0621391fb\example.vcxproj]
aws-cpp-sdk-core.lib(URI.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MT_StaticRelease' doesn't match value 'MD_DynamicRelease' in example.obj [C:\Users\marti\dev\clo ne\conan-center-index\recipes\aws-sdk-cpp\all\test_package\build\ecbfb49a4e2ea8618319be5f9e02c8e0621391fb\example.vcxproj]
aws-cpp-sdk-core.lib(DateTimeCommon.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MT_StaticRelease' doesn't match value 'MD_DynamicRelease' in example.obj [C:\Users\ma rti\dev\clone\conan-center-index\recipes\aws-sdk-cpp\all\test_package\build\ecbfb49a4e2ea8618319be5f9e02c8e0621391fb\example.vcxproj]
...
```

Reply:
This has something to do with mixing shared and static c runtime libraries.
Check https://github.com/aws/aws-sdk-cpp/blob/925f6015b700e7605482ca909601e0dd78624476/cmake/compiler_settings.cmake#L79-L87.

When building a static aws-sdk-cpp and shared runtime (MD), you need to pass `FORCE_SHARED_CRT`.

Reply:
After reading [the documentation](https://github.com/aws/aws-sdk-cpp/blob/master/Docs/CMake_Parameters.md#force_shared_crt) I rather do this change: 0f7de472

Does it looks find for you? It works on my local windows machine.